orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/OrchSessionItem.java

```diff
@@ -12,6 +12,7 @@ public class OrchSessionItem {
     public static final String ATTRIBUTE_VERIFIED_MFA_METHOD_TYPE = "VerifiedMfaMethodType";
     public static final String ATTRIBUTE_IS_NEW_ACCOUNT = "IsNewAccount";
     public static final String ATTRIBUTE_INTERNAL_COMMON_SUBJECT_ID = "InternalCommonSubjectId";
+    public static final String ATTRIBUTE_IS_AUTHENTICATED = "IsAuthenticated";
```

Comment:
I think up til now we've always called it `Authenticated`, any reason to change?

Reply:
Eh no real reason, can rename it